MobiFlight/Joysticks/Joystick.cs

```diff
@@ -409,6 +455,19 @@ public void SetOutputDeviceState(string name, string value)
 
         protected virtual void SendData(byte[] data)
         {
+            // Don't try and send data if no outputs are defined.
+            if (definition?.Outputs == null || definition?.Outputs.Count == 0)
+            {
+                return;
+            }
```

Comment:
Check added so there's no issue if a json file doesn't define `Outputs`, which takes care of devices that don't have any LEDs.